### PR TITLE
implemented support for custom components to be used in place of the …

### PR DESCRIPTION
**Pull request by tpizza**
@bebraw Here's my implementation for passing a custom component through to Field in searchtabular.

### CODE REVIEW

**Comment by git-toni** [comment]
@tpizza thanks for the PR, I'll wait for @bebraw 's opinion before merging. I've been a bit busy with other things and I'm not sure how much he's using `searchtabular` in general for reactabular` :)

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,5 @@ node_modules/
 coverage/
 dist/
 .eslintcache
+.idea
```
Comment (bebraw) [inline .gitignore]: Can you drop this ignore? It's too specific to belong here.

**Review by bebraw — CHANGES_REQUESTED** [review]
Can you add the new option to the readme as well? Looking good otherwise.

**Comment by tpizza** [comment]
@bebraw IntelliJ ignore has been removed.

**Comment by tpizza** [comment]
Ahhh, I just saw the request for the readme update.  Will make that revision now.

**Comment by bebraw** [comment]
Published as `1.3.0`. Thanks.

**Comment by git-toni** [comment]
@bebraw Oh, now I see mixed `selectabular`(which I collaborated a bit with) with `searchtabular`, you must have thought I'm crazy : )

**Comment by bebraw** [comment]
@git-toni No probs.

**Pull request merged**